### ISSUE TITLE
Steps for Max Moving Distance and Max still distance range

### ISSUE THREAD

**Issue by bednim** [rain931215/ESPHome-LD2410#9]
Hi, currently steps for max moving distance and range are every 1 step. is it possible to change it that steps are 0.1? when I change steps settings in esphome config and then adjust range to for example 2.5 - it will change itself down to 2; 3.5 to 3 etc etc. 

Is this possible at all?

Thank you

**Comment by rainchi** [comment]
No, the chip only support interger step, and I cannot change it's behavior.
